<!-- frobit launch file -->

<launch>
	<group ns="fmDecisionMakers">
		<node pkg="frobit_wptnav_mission" name="wptnav_mission" type="wptnav_mission_node.py" output="screen">
			<param name="remote_control_pub" value="/fmHMI/remote_control"/>
			<param name="remote_control_feedback_sub" value="/fmHMI/remote_control_feedback" />			
			<param name="automode_pub" value="/fmPlan/automode" />
	    		<param name="deadman_pub" value="/fmSafe/deadman" />
			<param name="cmd_vel_pub" value="/fmCommand/cmd_vel" />
			<param name="max_linear_velocity" value="50.7"/>
			<param name="max_angular_velocity" value="50.5"/>
		</node>
	</group>
</launch>

